### PR TITLE
dev_Yok

### PR DESCRIPTION
**Pull request by sRubyy**
This is my new feature.

### CODE REVIEW

File: YokFeature.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class YokFeature {
+    public void feature(){
+        // this is a new feature
```
Comment (tnktian) [inline YokFeature.java]: let's continued!

File: YokFeature.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+public class YokFeature {
+    public void feature(){
```
Comment (plphxwxt) [inline YokFeature.java]: gogo

**Comment by arthasem** [comment]
<html>
<body>


(・∀・)ノ
--



</body>
</html>

File: YokFeature.java (hunk 2 above)
Comment (YunduoLuo) [inline YokFeature.java]: nice work dude:)

File: lab-exercise-1/YokFeature.java [hunk 3]
```diff
@@ -1,5 +1,7 @@
 public class YokFeature {
     public void feature(){
         // this is a new feature
+        int x = 0;
+        x = x+1;
```
Comment (tnktian) [inline lab-exercise-1/YokFeature.java:5]: great job boi!

**Review by plphxwxt — COMMENTED** [review]
So good na man

**Review by YunduoLuo — COMMENTED** [review]
niceeeeeeeeeeeeee.

File: YokFeature.java (hunk 1 above)
Comment (arthasem) [reply YokFeature.java]: Good job bruh

**Pull request merged**